apps/artemis-web/pages/api/auth/signup.ts

```diff
@@ -45,7 +45,20 @@ const handler = nc()
         role: 'pending',
         token: '',
       })
-      .then(({ ops }) => ops[0]);
+      .then(({ insertedId }) => {
+        return insertedId;
+      });
+
+    const user = {
+      _id: nanoid(12),
```

Comment:
shouldn't this take the `user_id` value that was extracted by the previous call?

Reply:
Nice catch!! Basically, I saved the id to a temp var now, since I am not totally sure what this call is returning